workflow/workflow-demos-test/src_test/test/bpm/TestExpriy.java

```diff
@@ -0,0 +1,107 @@
+package test.bpm;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+
+import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
+import ch.ivyteam.ivy.bpm.engine.client.ExecutionResult;
+import ch.ivyteam.ivy.bpm.engine.client.element.BpmElement;
+import ch.ivyteam.ivy.bpm.engine.client.element.BpmProcess;
+import ch.ivyteam.ivy.bpm.exec.client.IvyProcessTest;
+import ch.ivyteam.ivy.workflow.CaseState;
+import ch.ivyteam.ivy.workflow.ITask;
+import ch.ivyteam.ivy.workflow.TaskState;
+import ch.ivyteam.ivy.workflow.WorkflowPriority;
+
+@IvyProcessTest
+public class TestExpriy
+{
+  private static final BpmProcess PROCUREMENT_PROCESS = BpmProcess.name("Expiry");
+  
+  private static final BpmElement EXPIRY_HIGHER_PRIO = PROCUREMENT_PROCESS.elementName("createExpriyTaskHigherPrio.ivp");
+  private static final BpmElement EXPIRY_OTHER_ROLE = PROCUREMENT_PROCESS.elementName("createExpiryTaskOtherRole.ivp");
+  private static final BpmElement EXPIRY_ERROR_TASK = PROCUREMENT_PROCESS.elementName("createExpriyErrorTask.ivp");
+  
+  @Test
+  void runProcessWithoutExpire(BpmClient bpmClient)
+  {
+    ExecutionResult result = startProcess(bpmClient, EXPIRY_HIGHER_PRIO);
+    bpmClient.start().anyNextTask(result).execute();
+    assertCaseIsDone(result);
+  }
+  
+  @Test
+  void expriyWithHigherPrio(BpmClient bpmClient)
+  {
+    ExecutionResult result = startProcess(bpmClient, EXPIRY_HIGHER_PRIO);
+    
+    ITask nextTask = getNextTask(result);
+    expireTask(nextTask);
```

Comment:
I would like to see an assert that the priority is normal before expireTask so that we see in the test what changes